rustup-init.sh

```diff
@@ -15,14 +15,13 @@
 
 set -u
 
-if [ -z "$RUSTUP_UPDATE_ROOT" ]; then
-    RUSTUP_UPDATE_ROOT="https://static.rust-lang.org/rustup"
-fi
+RUSTUP_UPDATE_ROOT=${RUSTUP_UPDATE_ROOT:-"https://static.rust-lang.org/rustup"}
 
 #XXX: If you change anything here, please make the same changes in setup_mode.rs
+version='rustup-init 1.16.0 (beab5ac2b 2018-12-06)'
```

Comment:
Should we keep this local to `usage`?

Reply:
Maybe rename it to `rustup_version`?

Reply:
The reason I split it to `version` is to easy to spot this need to be updated. People seems to forget to update this field.

Reply:
Ah, good reason.  I'm fine with it staying as "version".